src/components/Board/Board.component.js

```diff
@@ -298,6 +377,75 @@ export class Board extends Component {
             'is-locked': this.props.isLocked
           })}
         >
+          {isLocked && (
+            <Joyride
+              callback={data => {
+                const { status } = data;
+                if ([STATUS.FINISHED, STATUS.SKIPPED].includes(status)) {
+                  this.setState({ runLiveUnlock: false });
+                }
+              }}
+              steps={firststep}
+              continuous={true}
+              showSkipButton={true}
+              showProgress={true}
+              run={runLiveUnlock}
+              styles={{
+                options: {
+                  // modal arrow and background color
+                  arrowColor: '#eee',
+                  backgroundColor: '#eee',
+                  //button color
+                  primaryColor: '#aa00ff',
+                  //text color
+                  textColor: '#333',
+
+                  //width of modal
+                  width: 500,
+                  //zindex of modal
+                  zIndex: 1000
+                }
+              }}
+              locale={{
+                last: 'End tour',
+                skip: 'Close tour'
+              }}
+            />
+          )}
+          {!isLocked && (
```

Comment:
This is just a suggestion, but since most of the code is the same whether the board is locked or not, you could define `Joyride` once and use ternary expressions. Something like:

```javascript
<Joyride
  callback={data => {
    const { status } = data;
    if ([STATUS.FINISHED, STATUS.SKIPPED].includes(status)) {
      this.setState(
        isLocked ? { runLiveUnlock: false } : { runLiveHelp: false }
      );
    }
  }}
  steps={isLocked ? firststep : steps}
  run={isLocked ? runLiveUnlock : runLiveHelp}
  ...
/>
```

I'm not a frontend engineer though so let's see what the experts say first :-)

Reply:
I believe you are right. Because of the fact that we are not so familiar yet with JavaScript and React we took it step by step, but if this is better for the code we will change it of course!